grapher/core/Grapher.tsx

```diff
@@ -1218,8 +1218,8 @@ export class Grapher
     // All columns that are necessary for the current view, including meta columns like entityName
     // and time
     @computed get columnSlugsNecessaryForCurrentView() {
-        const annotationSlugs = this.activeColumnSlugs.map((slug) =>
-            this.inputTable.getAnnotationColumnSlug(this.inputTable.get(slug))
+        const annotationSlugs = this.activeColumnSlugs.map(
+            (slug) => this.inputTable.getAnnotationColumnForColumn(slug)?.slug
```

Comment:
Ok I think I got it! 

The difference in logic in this commit I think is: instead of `this.inputTable.get(slug)` it does `this.inputTable.get(slug).def`. And only the `def` contains the correct annotations column slug, while the column itself doesn't. 

It's really not great that the metadata is duplicated, but _not quite fully_. A symptom that something is going bad somewhere else, and also that we shouldn't duplicate metadata but have a "single source of truth".